crates/gpu/src/multi_buffer.rs

```diff
@@ -410,6 +481,17 @@ impl<T: bytemuck::Pod> TypedMultiBuffer<T> {
         offset: u64,
         data: &[T],
     ) -> Result<(), MultiBufferError> {
+        // TODO: correctly update len for the written subbuffer
```

Comment:
Does this need to be dealt with for the merge?

Reply:
I couldn't do it as well since only `&self` is available, and this would need `&mut self`.